dev/devfile.yaml

```diff
@@ -17,19 +17,28 @@ components:
     memoryLimit: 1512Mi
     type: dockerimage
     alias: appsodyrun
-    image: 'ajymau/java-openliberty-dev:latest'
+    image: 'kabanero/ubi8-maven:0.3.1'
     env:
       - value: TEST2
         name: MODE2
       - value: myval2
         name: myprop2
+    volumes:
+      - name: m2
+        containerPath: /root/.m2/repository
 apiVersion: 1.0.0
 commands:
+  - name: devInit
+    actions:
+      - workdir: /projects/user-app
+        type: exec
+        command: mvn -B -DskipTests=true liberty:install-server liberty:create liberty:install-feature install dependency:go-offline 
+        component: appsodyrun 
   - name: devRun
     actions:
       - workdir: /projects/user-app
         type: exec
-        command: mvn -DhotTests=true -DappsDirectory=apps -Dmaven.repo.local=/mvn/repository liberty:dev
+        command: mvn -DhotTests=true -DappsDirectory=apps liberty:dev
```

Comment:
If we go back and read through: https://github.com/OpenLiberty/ci.maven/issues/718
it seems like we can lose the "-DappsDirectory=apps"   since now our first liberty:create will have been done with the server.xml already present. 

Instead of trying to outsmart liberty-maven-plugin here let's just do what's simple, better aligning with what the average user would try... so just do:  `mvn -DhotTests=true liberty:dev`